src/components/VNoticeBar/meta/NoticeBar.stories.mdx

```diff
@@ -10,10 +10,12 @@ import NoticeBar from '~/components/VNoticeBar/VNoticeBar'
 
 <Meta title="Components/Notice bar" component={NoticeBar} />
 
-export const Template = (args, { argTypes }) => ({
-  template: `<NoticeBar>{{ text }}</NoticeBar>`,
+export const Template = (args) => ({
+  template: `<NoticeBar>{{ args.text }}</NoticeBar>`,
   components: { NoticeBar },
```

Comment:
```suggestion
  template: `<VNoticeBar>{{ args.text }}</VNoticeBar>`,
  components: { VNoticeBar },
```
Can we rename the component and the file here?

Reply:
Good catch! Sure we can fix this. Made the change in 360a6e1.